src/tools/rz-diff/binary_diffing.md

```diff
@@ -1,75 +1,181 @@
-                          fcn.00003a50  120 0x3a50 | UNMATCH  (0.125000) | 0x3a60   120 fcn.00003a60
+--- /bin/true
++++ /bin/false
+@@ -1,3 +1,3 @@
+-virt: 0x00000000000026f0 phys: 0x00000000000026f0 entry init
+-virt: 0x00000000000026b0 phys: 0x00000000000026b0 entry fini
+-virt: 0x0000000000002610 phys: 0x0000000000002610 entry program
++virt: 0x0000000000002700 phys: 0x0000000000002700 entry init
++virt: 0x00000000000026c0 phys: 0x00000000000026c0 entry fini
++virt: 0x0000000000002620 phys: 0x0000000000002620 entry program
+
 ```
 
-And now a cool feature : rizin supports graph-diffing, à la [DarunGrim](http://www.darungrim.org/), with the `-g` option. You can either give it a symbol name, of specify two offsets, if the function you want to diff is named differently in compared files. For example, `rz-diff -md -g main /bin/true /bin/false | xdot -` will show differences in `main()` function of Unix `true` and `false` programs. You can compare it to `rz-diff -md -g main /bin/false /bin/true | xdot -` (Notice the order of the arguments) to get the two versions.
-This is the result:
+### Diffing fields in Binaries:
 
-![/bin/true vs /bin/false](img/true_false2.png)
+`rz-diff -t fields /bin/true /bin/false `
 
-Parts in yellow indicate that some offsets do not match. The grey piece means a perfect match. The orange one highlights a strong difference. If you look closely, you will see that the left part of the picture has `mov eax, 0x1; pop rbx; pop rbp; ret`, while the right one has `xor edx, edx; pop rbx; pop rbp; ret`.
+```
+--- /bin/true
++++ /bin/false
+```
 
-Binary diffing is an important feature for reverse engineering. It can be used to analyze [security updates](https://en.wikipedia.org/wiki/Patch_Tuesday), infected binaries, firmware changes and more...
+### Diffing sections in Binaries:
 
-We have only shown the code analysis diffing functionality, but rizin supports additional types of diffing between two binaries: at byte level, deltified similarities, and more to come.
+`rz-diff -t sections /bin/true /bin/false`
+```
 
-We have plans to implement more kinds of bindiffing algorithms into rizin, and why not, add support for ASCII art graph diffing and better integration with the rest of the toolkit.
+--- /bin/true
++++ /bin/false
+```
+### Diffing strings in Binaries:
 
+```rz-diff -t strings /bin/true /bin/false
+--- /bin/true
++++ /bin/false
+@@ -11,7 +11,7 @@
+ Written by %s, %s, %s,\n%s, %s, %s, %s,\nand %s.\n
+ Copyright %s %d Free Software Foundation, Inc.
+       --help     display this help and exit\n
+-Exit with a status code indicating success.
++Exit with a status code indicating failure.
+ Written by %s, %s, %s,\n%s, %s, %s, and %s.\n
+ Written by %s, %s, %s,\n%s, %s, and %s.\n
+ https://www.gnu.org/software/coreutils/
+@@ -51,7 +51,7 @@
+ --help
+ ASCII
+ POSIX
++false
+ shell
+ %s\n\n
+ 8.30
+-true
+
+```
+### Diffing symbols in Binaries:
+
+```rz-diff -t symbols /bin/true /bin/false
+--- /bin/true
++++ /bin/false
+```
```

Comment:
Could you please add also a few paragraphs about splitview hexadecimal diffing mode? See `rz-diff -H` and https://github.com/rizinorg/rizin/pull/1148